.github/workflows/deploy.yml

```diff
@@ -221,8 +250,8 @@ jobs:
            echo ::set-output name=new_tag::"PR-${TAG}"
            echo ::set-output name=changelog::"${BODY}"
 
-       - name: Create a GitHub release
-         if: github.ref == 'refs/heads/master' && github.event.inputs.environment == 'Development' && steps.tag_version.outputs.new_tag != '0'
+       - name: Create a GitHub Pre-Release
```

Comment:
Looking lower down this is creating a draft release I think, rather than a 'pre-release' release ? Probably should be renamed accordingly?

Reply:
Agreed